### ISSUE TITLE
v0.8.0 does not work with ESLint v8.x

### ISSUE THREAD

**Issue by brandonb927** [ihordiachenko/eslint-plugin-chai-friendly#35]
There appears to be an issue with v0.8.0 that is not present in 0.7.4, likely introduced by the ESLint v9 format support inclusion:

```
Oops! Something went wrong! :(

ESLint: 8.57.0

TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Object'
    |     property 'chai-friendly' -> object with constructor 'Object'
    |     property 'configs' -> object with constructor 'Object'
    |     property 'recommended' -> object with constructor 'Object'
    --- property 'plugins' closes the circle
Referenced from: /home/runner/work/.eslintrc.js
    at JSON.stringify (<anonymous>)
    at /home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2156:45
    at Array.map (<anonymous>)
    at ConfigValidator.formatErrors (/home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2147:23)
    at ConfigValidator.validateConfigSchema (/home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2177:84)
    at ConfigArrayFactory._normalizeConfigData (/home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3019:19)
    at ConfigArrayFactory._loadExtendedPluginConfig (/home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3239:25)
    at ConfigArrayFactory._loadExtends (/home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3154:29)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/runner/work/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:3095:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
```

I am using the `extends` key in the legacy `.eslintrc` format as such:
```json
{
  "extends": ["plugin:chai-friendly/recommended"]
}
```

**Comment by MikeMcC399** [comment]
@brandonb927 

I can reproduce your issue.

**Comment by ihordiachenko** [comment]
Hi, @brandonb927. Thanks for the report, I will look into it this week.

If you use ESLint v8 with an `eslintrc` file, you can safely downgrade to v0.7.4 for now. The only significant difference between 0.7.x and 0.8.x is the ESLint v9 API and flat config support

**Comment by brandonb927** [comment]
> Hi, @brandonb927. Thanks for the report, I will look into it this week.
> 
> If you use ESLint v8 with an `eslintrc` file, you can safely downgrade to v0.7.4 for now. The only significant difference between 0.7.x and 0.8.x is the ESLint v9 API and flat config support

Does that mean that 0.8.0 is a breaking change to support ESLint v9?

**Comment by MikeMcC399** [comment]
@brandonb927 

> Does that mean that 0.8.0 is a breaking change to support ESLint v9?

I would assume that it is just a bug at this stage. If it were an intentional breaking change, then it would need to be a major version change to `1.0.0`

**Comment by brandonb927** [comment]
I would think that as well @MikeMcC399. We'll stay on 0.7.4 for now. Thanks folks!

**Comment by ihordiachenko** [comment]
@brandonb927 fixed in `eslint-plugin-chai-friendly@1.0.0`. No changes in your `eslintrc` needed. The fix required a breaking change for the flat config users, hence the major version bump

**Comment by brandonb927** [comment]
Thank you @ihordiachenko! We ended up hitting this issue because Dependabot bumped the version so we ignored it, I will test the new 1.0.0 version soon.
